packages/wallets/src/features/cfd/modals/MT5TradeModal/MT5TradeModal.tsx

```diff
@@ -1,10 +1,23 @@
-import React from 'react';
+import React, { FC, useEffect } from 'react';
+import { useModal } from '../../../../components/ModalProvider';
+import { TMarketTypes, TPlatforms } from '../../../../types';
 import { ModalTradeWrapper } from '../../components';
 import { MT5TradeScreen } from '../../screens';
 
-const MT5TradeModal = () => {
+type TMT5TradeModal = {
+    marketType?: TMarketTypes.All;
+    platform: TPlatforms.All;
+};
+
+const MT5TradeModal: FC<TMT5TradeModal> = ({ marketType, platform }) => {
+    const { setModalState } = useModal();
+    useEffect(() => {
+        setModalState('marketType', marketType);
+        setModalState('platform', platform);
+    }, [marketType, platform]);
```

Comment:
Missing dependency ? 🤔

Reply:
I wish these appeared in my vscode, I only can see them when I pushed my PR 😮‍💨